config/nudge_port.txt

```diff
@@ -0,0 +1 @@
+45678
```

Comment:
this should have a more descriptive name like

server_to_tool_bridge_port.txt

characters are mostly free